lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -1,1767 +1,191 @@
-                padding-bottom: 50px;
-                overflow: hidden;
-                position: relative;
-            }
+  .section_title {
+    margin-bottom: $baseline;
+    color: theme-color("dark");
+  }
 
-            .tbl-view {
-                .tbl-col {
-                    .btn-enroll-now {
-                        position: absolute;
-                        left: 0;
-                        right: 0;
-                        bottom: 0;
-                        width: 100%;
-                        border-radius: 0;
-                        line-height: 22px;
-                        background: #0f8012;
-                        @media (min-width: 768px) {
-                            position: relative;
-                            border: 1px solid #ccc;
-                            max-width: 200px;
-                        }
-                    }
+  .thick_rule {
+    height: 4px;
+    border: 0;
+    background-color: theme-color('secondary');
+  }
 
-                    &:first-child {
-                        @media (min-width: 768px) {
-                            width: 75%;
-                        }
+  .main-banner {
+    color: white;
+    margin-bottom: 1px;
 
-                        p {
-                            @media (min-width: 768px) {
-                                max-width: 700px;
-                            }
-                        }
-                    }
-                }
-            }
-            .btn-enroll-now{
-                width: 100% !important;
-                max-width: 300px !important;
-                min-height: 50px;
-                font-size: 1.33rem;
-                font-weight: bold;
-            }
-        }
-
-        .pagingInfo {
-            display: block;
-            text-align: center;
-            @media (min-width: 768px) {
-              display: none;
-            }
-        }
-    }
-
-    .copy-meta-mobile{
-      font-size: 0.875rem;
-      line-height: 1.6em;
+    .authoring-org-logo {
+      background-color: $white;
+      margin-bottom: $baseline;
     }
 
-    //Helper Classes
-    .pull-left{
-      float: left !important;
+    .btn {
+      font-size: 20px;
     }
 
-    .pull-right{
-      float: right !important;
+    .btn,
+    .authoring-org-logo,
+    h1,
+    h2,
+    a {
+      margin-bottom: $baseline;
     }
 
-    .block{
-      display: block;
-    }
+    #program_video {
+      display: flex;
+      align-items: center;
+      height: 100%;
 
-    a{
-      text-decoration: none;
-      &:hover{
-        opacity: 0.8;
-        text-decoration: none;
-      }
-      &:focus{
-        text-decoration: none;
-      }
-    }
-    .no-padding{
-      padding: 0 !important;
-    }
-    #success-message{
-      padding: 0;
-      .alert {
-        min-width: 0;
-        border-left-width: 0;
-        border-right-width: 0;
-        border-bottom-width: 0;
-        box-shadow: none;
+      button {
+        background-color: rgba(0, 0, 0, 0);
```

Comment:
Are you trying to make this a transparent background? If so you can just use background-color: transparent as opposed to setting it to a 0 opacity black

Reply:
Ahh, yep, that was the goal. It's been a while since I've dealt with front-end. Fixed.